src/pages/InflectPage/components/Pali1AutoComplete/Pali1AutoComplete.tsx

```diff
@@ -34,15 +34,17 @@ export const Pali1AutoComplete = ({ db, initialValue, onChangePali1 }: Pali1Auto
     }
 
     const loadOptions = () => {
-      const results = db.exec(

```

Comment:
not the correct fix. there is a marquee "loading db" already - meaning user should not be able to enter anything till the db is loaded.

can you think through the scenarios and overall app design before making the changes? it appears we are making the first change that comes to mind instead of having at least 2 to choose from.

Reply:
did consider that. two reasons why I hadn't considered it - 
1. solution would involve another set of UI control logic ie check status of download & unzip operation + disable an input element based on it. the present sol needs no change wrt MLab specific code. 
2. Isn't being able to enter the query right after the page loads rather than having to wait for a couple of seconds more (for the db) favourable from a general user PoV? esp since the integrity of the search functionality remains the same both ways.

Reply:
the best experience for the user is to tell the user what is going on and prevent them from getting into states that code cannot handle.

why would we need new controls? there is also a db null logic for the generated html - just move it up to cover the input box as well.

Reply:
Disabled the entire autocomplete component instead of just the input box since it'd still load options if I did the latter.
disabled={!db}